new/LOC7418MammoHela.xml

```diff
@@ -0,0 +1,58 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LOC7418MammoHela" xml:lang="en" type="place">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Māmmo Ḥǝlā</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-01-09">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="area">
+                    <placeName xml:lang="gez" xml:id="n1">ማሞ፡ ሕላ፡</placeName>
```

Comment:
It is very plausible that it could also be a personal name, e.g. the land that (previously) belonged to Māmmo Ḥǝlā, who might have been a tenant or an important person who had settled this land in the past. This can also be argued for Ṣāmǝtā and Sāzā (or Mǝsāzā). These names are not attested as either place names or personal names. As far as I can remember, Māmmo is a popular personal name, but I have not come across it in connection with Ḥǝlā. I do not know whether it is most appropriate to encode these names as place names or personal names, or to leave them unmarked. This is a regular ambivalence that puzzles me in many additions, especially in the case of land grants.

Here is the addition as I have prepared it:
```
<item xml:id="a2">
                           <locus from="1va" to="1vb"/>
                           <desc type="LandGrant">Record of a land grant by ʿaqqābe saʿat <persName ref="PRS14065Tewogolos"/>
                              to marāḥi <persName ref="PRS9199TaklaYo">Takla Yoḥannǝs</persName> in compensation for the 
                              land that had been troyed during the reign of <persName ref="PRS6229LebnaDe"/>.</desc>
                           <q xml:lang="gez">በአኰቴተ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ወሀብኩ፡ አነ፡ ዓቃቤ፡ ሰዓት፡ 
                              <persName ref="PRS14065Tewogolos">ቴዎጎሎስ፡</persName> 
                              ምድረ፡ ፩ዘ<placeName ref="LOC7418MammoHela">ማሞ፡ ሕላ፡</placeName> ዘነበረ፡ እንዘ፡ ይበልዕ፡ አበዛ፡ የኔኩን፡ 
                              ወ፩፡ እምደዳ፡ ዘ<placeName ref="LOC7419Tsamita">ጻምታ፡</placeName> 
                              ወ፩እምሀገረ፡ ሰላም፡ እም<placeName ref="LOC7420Saza">ሳዛ፡</placeName> ምድረ፡ ዘቤተ፡ ንጉሥ፡ እሎንተ፡ ፫ገራውሀ፡ ወሀብኩ፡ ለመራሂ፡ 
                              ለ<persName ref="PRS9199TaklaYo">ተክለ፡ ዮሐንስ፡</persName> ሶበ፡ ነስኡ፡ ወሜጡ፡ ምድሮ፡ ፫መንገለ፡ ጕልተ፡ መካነ፡ ስላሤ፡ አእሚርየ፡ ከመ፡ 
                              ግብረ፡ ምርሃት፡ ጽኑዕ፡ ውእቱ፡ ዘንተ፡ ኢይንስእዎ፡ ለእመ፡ ተመይጠሂ፡ ሎቱ፡ ም<cb n="1vb"/>ድሩ፡ ወለእመ፡ ኢተመይጠሂ፡ እስመ፡ ምድረ፡ ምርሃት፡ ዘዓቃቤ፡
                              ሰዓት፡ ሕፁጽ፡ ውእቱ፡ እመቦ፡ ዘሄዶ፡ ወተዳገሎ፡ <gap reason="ellipsis" unit="lines" quantity="5"/>
                              ወበ<add place="margin">አፈ፡</add> <persName ref="PRS5633IyasusM">ኢየሱስ፡ ሞአ፡</persName> ለእመ፡ ይቤ፡ ተዓዲውየ፡
                              እነሥኦ፡ እለተ፡ ሞቱ፡ ፍጡነ፡ ይምጽኦ፡ ከርሠ፡ መቃብር፡ ለበድኑ፡ ይገፍትኦ፡ አራዊት፡ ወአዕዋፍ፡ ከመ፡ ይብልዖ፡ አማን፡ ዘንተ፡ መጽሐፈ፡ የደምሰሰ፡ 
                              ወ<add place="above">ዘ</add>ፈሐቀ፡ እምጸጋ፡ ጥምቀት፡ ይኩን፡ ዕሩቀ፡ በአፍትኖ፡ ካህን፡ ወበነሢአ፡ ቍርሳን፡ እይክሀል፡ ጸዲቀ፡ አሜን፡ ወአሜን፡ ለይኩን። ።
                           </q>
                        </item>
```

<img width="472" alt="Screenshot 2024_MammoHela" src="https://github.com/BetaMasaheft/Places/assets/40291787/7b6bf46a-a4eb-496a-9b52-bca335d49a24">